scripts/update-sdk.sh

```diff
@@ -17,9 +17,16 @@
 set -euo pipefail
 
 LATEST_SDK_TAG=$(curl -sSfL -X GET  https://api.github.com/repos/mongodb/atlas-sdk-go/releases/latest | jq -r '.tag_name')
-
 LATEST_SDK_RELEASE=$(echo "${LATEST_SDK_TAG}" | cut -d '.' -f 1)
-echo  "==> Updating SDK to latest major version ${LATEST_SDK_TAG}"
+
+echo  "==> Updating SDK to latest major version tag: ${LATEST_SDK_TAG}, release: ${LATEST_SDK_RELEASE}"
+
+echo "Refreshing cache info and waiting 60s"
+curl -sSfL -X GET  "https://proxy.golang.org/go.mongodb.org/atlas-sdk/${LATEST_SDK_RELEASE}/@v/${LATEST_SDK_TAG}.info"
```

Comment:
```suggestion
curl -sSL -X GET  "https://proxy.golang.org/go.mongodb.org/atlas-sdk/${LATEST_SDK_RELEASE}/@v/${LATEST_SDK_TAG}.info"
```
just in case, I'm not sure if this will fail the first time and that may be ok

Reply:
you're right, at least the cache ask is kicked, so if you retry later it'll probably succeed.  should i have more than 60s to start with?